components/brave_new_tab_ui/stories/default.tsx

```diff
@@ -19,8 +19,27 @@ import * as binanceActions from '../actions/binance_actions'
 import * as geminiActions from '../actions/gemini_actions'
 import * as bitcoinDotComActions from '../actions/bitcoin_dot_com_actions'
 import * as cryptoDotComActions from '../actions/cryptoDotCom_actions'
+import * as todayActions from '../actions/today_actions'
 import store from '../store'
 import { getNewTabData, getGridSitesData } from './default/data/storybookState'
+import getTodayState from './default/data/todayStorybookState'
+// Uncomment to use actual images proxied from a CORS-breaker proxy
+// TODO(petemill): privateCDN should be in /common/
+// import { getUnpaddedAsDataUrl } from '../../brave_extension/extension/brave_extension/background/today/privateCDN'
+
+// TODO(petemill): Have the private CDN contain a CORS response so we can directly fetch
```

Comment:
Just to be clear, this is a storybook page meaning that this code is not bundled and delivered to users. It runs on dev machines and a static website, as a development and test tool. So the mentions of using this random 3rd party `cors-anywhere` site should be temporary and never for production code.

Reply:
Fair enough. I also don't think that it's a problem for the private CDN to return CORS headers since everything hosted there is public and credential-less.